models/grading/fact_grade_status.sql

```diff
@@ -0,0 +1,64 @@
+with
+    latest_grade_status as (
+        select
+            org, course_key, object_id, actor_id, max(emission_time) as last_grade_time
+        from {{ ref("grading_events") }}
+        where
+            verb_id in (
+                'http://adlnet.gov/expapi/verbs/passed',
+                'http://adlnet.gov/expapi/verbs/failed'
+            )
+        group by org, course_key, object_id, actor_id
+    ),
```

Comment:
I don't think we need this CTE if we're also using a window function below to get the most recent record.